EXAMPLE/Cycle-Audio-Devices.ps1

```diff
@@ -0,0 +1,30 @@
+# This script cycles through all audio devices
+
+
+# Get a list of all enabled devices
+[System.Collections.ArrayList]$audioDevicesList = Get-AudioDevice -List
+
+# Get the default playback device as <AudioDevice>
+$defaultPlaybackDevice = Get-AudioDevice -Playback
+
+#Remove non-playback devices from audioDevicesList
+for ( $index = 0; $index -lt $audioDevicesList.count; $index++ ) {
+   if ($audioDevicesList[$index].type -ne "Playback") {
+      $audioDevicesList.Remove($audioDevicesList[$index])
+   }
+}
+
+#Loop through audioDevicesList
+for ( $index = 0; $index -lt $audioDevicesList.count; $index++ ) {
+   #Find the position of the default audio device
+   if ( $audioDevicesList[$index].name -eq $defaultPlaybackDevice.name) {
```

Comment:
The fact that different devices can have the same name can cause this line to not work as it should. I would suggest something like replacing _.name_ by _.ID_